content/it/docs/reference/glossary/kube-controller-manager.md

```diff
@@ -0,0 +1,18 @@
+---
+title: kube-controller-manager
+id: kube-controller-manager
+date: 2018-04-12
+full_link: /docs/reference/command-line-tools-reference/kube-controller-manager/
+short_description: >
+  Componente sui master node che gestisce i controller.
```

Comment:
Componente del control plane (same below)

Reply:
Sure, but fix that in the English locale first.